Gruntfile.js

```diff
@@ -2,11 +2,14 @@ module.exports = function(grunt) {
     grunt.initConfig({
         eslint: {
             all: ['*.js', './public/js/*']
+        },
+        flake8: {
+            src: ['*.py', './']
```

Comment:
`'*.py'` and `'./'` both flake .py files in root only I believe. Change to `['*.py', './util/*.py']` after #30 is in. You can also do `['**/*.py']` which will flake ALL .py files in the project[1]. 
[1] https://www.npmjs.com/package/grunt-flake8#default-options